Assets/MRTK/SDK/Experimental/PulseShader/Scripts/HideOnDevice.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HideOnDevice : MonoBehaviour
```

Comment:
This feels somewhat like an example script, or at least not an SDK script that's specific to the pulse shader.

Reply:
Moved to Examples folder.